.python-version

```diff
@@ -0,0 +1 @@
+Sel-Env-3.9.7
```

Comment:
This shouldn't be checked in.  Maybe add `.python-version` to `.gitignore` so others don't encounter it.